### ISSUE TITLE
Add localization as a way to contribute

### ISSUE THREAD

**Issue by emmairwin** [firefox-devtools/debugger#7327]
I could add a PR to the contributing file, but assume  it's a bigger discussion of 'what, how and where'.  In our D&I research, someone suggested they would like to contribute to localization of the 'Getting Started' content.  

cc /   @nmicht  who is a member of our cohort, and is already planning a PR to 'Getting Started' for English version.

**Comment by emmairwin** [comment]
This would be dependent on #7353 
@AnshulMalik talked a bit about how this *might* work (challenge that we keeping localized content up to date requires intention.   Doing some research to see how other projects handle this type of request.

**Comment by jasonLaster** [comment]
it would be nice to have a `getting-started-locale.md` If we have a couple we can ping folks after a doc change lands to update it. 

Also, github has the ability to add people as watchers of files so people could sign up to watch for changes to docs.

**Comment by emmairwin** [comment]
@jasonLaster that's really interesting to learn about the codeowner/pings.   

Also just for inspiration this project does a great job:  https://vuejs.org/v2/guide/installation.html

**Comment by janodvarko** [comment]
Firefox Developer Tools (including Debugger tool) development is now entirely happening in Bugzilla and new issues should be reported here: https://bugzilla.mozilla.org/enter_bug.cgi?product=DevTools&component=Debugger

Feel free to reopen this report in Bugzilla, thank you.

Honza
